src/util/removeTempRichmediaRcSync.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
```

Comment:
so now we have sync and async function for this. async is faster and is used in build where we don't have to close the server manually. sync is required for process.on('exit') since it can't run async code due to the nature of the event. Also, I made some improvements to the function itself so it's more reliable and snappy.

Reply:
didn't test the case you described but I didn't have any leftover tmp files during my tests

Reply:
The problem with suddenly closing a server is Windows. When the process is running and you Ctrl-C, windows executes 'kill' that is not preventable. And since the leftover files are still the problem I added a new mode `-m cleanup` to delete all such files. Ctrl-C still works when you're just listening to a port and not executing anything.